coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -1548,6 +1548,10 @@ ul {
 .actions__more-actions {
   padding: 0;
 
+  .v-list-item {
+    min-width: 160px;
```

Comment:
Looks like the menuitem could use some vertical centering also.
![drop down](https://user-images.githubusercontent.com/10002889/72007564-d3a1a480-
3206-11ea-9b22-3a240be8e691.PNG)

Reply:
Please also check v centering on others.